### PR TITLE
#67 - extended MultipartTest and set coverage metrics

### PR DESCRIPTION
**Pull request by olenagerasimova**
For #67 I've extended `MultipartTest` and set coverage metrics to 0.75.

### CODE REVIEW

File: src/test/java/com/artipie/pypi/http/MultipartTest.java [hunk 1]
```diff
@@ -67,11 +74,11 @@ void failsIfBoundaryIsNotProvided() {
             Assertions.assertThrows(
                 CompletionException.class,
                 () -> new Multipart(
-                    new Headers.From(new ContentType("123")), Flowable.empty()
+                    new Headers.From(new ContentType("098")), Flowable.empty()
```
Comment (olegmoz) [inline src/test/java/com/artipie/pypi/http/MultipartTest.java:77]: @olenagerasimova could you please explain what is the reason for this update? It does not seem to be related to the issue at hand

Comment (olenagerasimova) [reply src/test/java/com/artipie/pypi/http/MultipartTest.java]: @olegmoz it was first place to correct in order not to suppress `AvoidDuplicateLiterals` check. I've corrected it back and used another string in new test method

Comment (olegmoz) [reply src/test/java/com/artipie/pypi/http/MultipartTest.java]: @olenagerasimova ah, got it

**Comment by 0crat** [comment]
Job `gh:artipie/pypi-adapter#91` is not assigned, can't get performer
